website/docs/rbs-support.md

```diff
@@ -871,6 +867,60 @@ argument stays `Array`.
 
 ## Unsupported features
 
+### Abstract methods
+
+While the `@abstract` annotation is technically supported for methods, it is
+currently not recommended. To understand why, take the following example:
+
+```ruby
+# @abstract
+class Foo
+  # @abstract
+  # () -> void
+  def foo; end
+end
+
+class Bar < Foo
+  # @override
+  # () -> void
+  def foo
+    super
+  end
+end
+```
+
+When using traditional Sorbet sigs, the call to `super` inside of `Bar#foo`
+would error at runtime, because `sorbet-runtime` uses metaprogramming to
+remove the `Foo#foo` method. Any mistaken calls to this method would raise an
+error at runtime.
+
+However, RBS signatures have no runtime component. This means that any
+inadvertent calls to abstract methods will silently no-op, which has the
+potential to create subtle bugs and unintended behavior in our applications.
+
+Instead of using the `@abstract` annotation on methods, place any abstract
+methods in a separate `.rbi` shim file.
+
+Modifying the example above, place only the abstract class definition in a
+Ruby file:
+
+```ruby
+# @abstract
```

Comment:
Honestly at this point they could also move the class? 🤔

Reply:
Oh let me clarify the example and include Bar.

Reply:
Is the current example clearer? My intention was to keep the definition of `Foo` in the Ruby files because it has to exist at runtime.